index.js

```diff
@@ -1,18 +1,28 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector('#header')
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = "pink"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+const profile = document.querySelector('#profile') 
+const img = profile.querySelector('img')
+const name = profile.querySelector('h2')
+const nickname = profile.querySelector('em')
+img.src = traveler.photo
+img.alt = traveler.name
+name.textContent = traveler.name
+nickname.textContent = traveler.nickname
 
 
 /***** Deliverable 4 *****/
-
+// Sorry I didn't have enough time to work on the rest!
+// I would like to study on this a little more rather than
+// approach this question with random wrong answers!
 
 
 /***** Deliverable 5 *****/
```

Comment:
No problem! So far, your solutions look good! Feel free to finish later and resubmit. You're on the right track!